server/mongo.js

```diff
@@ -3,7 +3,7 @@ const env = require('./env/environment');
 
 mongoose.Promise = global.Promise;
 
```

Comment:
@SnehaGunda, are you sure this change is necessary? I didn't find it to be needed when building and running the code.

Reply:
Hey yes its Burke's suggestion, I had an email conversation with him

Reply:
I am closing this PR, seems like I didn't open it to the right target, will create a new one